python/ray/experimental/serve/tests/test_task_runner.py

```diff
@@ -2,7 +2,8 @@
 
 import ray
 import ray.experimental.serve.context as context
-from ray.experimental.serve.queues import CentralizedQueuesActor
+from ray.experimental.serve.queues import (RoundRobinPolicyQueueActor as
```

Comment:
Can you change the usage of CentralizedQueuesActor in this file to be RoundRobin? We don't need to alias it.